prow/github/client_test.go

```diff
@@ -1600,3 +1600,19 @@ func TestListMilestones(t *testing.T) {
 		t.Errorf("Didn't expect error: %v", err)
 	}
 }
+
+func TestListPRCommits(t *testing.T) {
+	ts := httptest.NewTLSServer(http.HandlerFunc(func(w http.ResponseWriter, r *http.Request) {
+		if r.Method != http.MethodGet {
+			t.Errorf("Bad method: %s", r.Method)
+		}
+		if r.URL.Path != "/repos/theorg/therepo/pulls/3/commits" {
+			t.Errorf("Bad request path: %s", r.URL.Path)
+		}
+	}))
+	defer ts.Close()
+	c := getClient(ts.URL)
+	if err, _ := c.ListPRCommits("theorg", "therepo", 3); err != nil {
```

Comment:
`err` should be the second value returned by `ListPRCommits` not the first.
Please augment this test to at least check that the expected number of commits is returned.